app/models/storage.rb

```diff
@@ -290,27 +290,20 @@ def self.scan_watchdog(miq_task_id)
     scan_queue_watchdog(miq_task.id)
   end
 
-  def self.vmdb_storage_config
-    VMDB::Config.new("storage").config
-  end
-
   DEFAULT_WATCHDOG_INTERVAL = 1.minute
   def self.scan_watchdog_interval
-    config = vmdb_storage_config
-    return DEFAULT_WATCHDOG_INTERVAL if config['watchdog_interval'].nil?
-    config['watchdog_interval'].to_s.to_i_with_method
+    return DEFAULT_WATCHDOG_INTERVAL if ::Settings.storage.watchdog_interval.nil?
+    ::Settings.storage.watchdog_interval.to_s.to_i_with_method
```

Comment:
btw, above settings are not in the ```settings.yml``` and were added to https://docs.google.com/a/redhat.com/spreadsheets/d/1byRsMvwmOxLSP-tvwAwqQH5Vw2shdcjjrdy4c7FHM2o/edit?usp=sharing